README.md

```diff
@@ -16,7 +16,7 @@ This library provides a simple abstraction around URLSession and HTTP. There are
     * Make error handling more pleasant.
     * Make it easy to define the details of your request and the model type you want to get back.
 * Keep things simple.
-    * There are currently around 600 SLOC, with about a third of that being boilerplate HTTP definitions.
+    * There are currently around 1300 SLOC, with about a quarter of that being boilerplate HTTP definitions.
```

Comment:
I should probably document this, but I've been measuring SLOC for the actual library itself (on the `Sources/Hyperspace` directory). This puts it at around 800 SLOC right now.
![image](https://user-images.githubusercontent.com/1720304/42950585-fe81d15e-8b39-11e8-9cb8-9b4249567e87.png)

Or would you rather us take into consideration the unit tests and example code for the line count?

Reply:
I apparently counted all lines. So I like your metric better. Will update.